ui_framework/components/table/table_row_check_box_cell.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { KuiTableRowCell } from './table_row_cell';
+
+export const KuiTableRowCheckBoxCell = ({ onChange, isChecked, className, ...rest }) => {
```

Comment:
Can we add a test file for this component, to verify that custom classes and other attributes can be supplied?